article/app/views/package.scala

```diff
@@ -36,7 +38,7 @@ object MainCleaner {
 object BodyCleaner {
   def apply(article: Article, html: String, amp: Boolean)(implicit request: RequestHeader) = {
     implicit val edition = Edition(request)
-    val cleaners = List(
+    val cleaners = ListBuffer(
```

Comment:
if you're going to have a mutable list, it's better if it's local, but the same arguments about being able to reason about your code apply regardless.  I'd recommend keeping it immutable.
Having said all that, there may be a better (but still functional) way of building up the list as a logic for whats in it becomes more complicated.